browser/src/map/Clipboard.js

```diff
@@ -708,28 +708,35 @@
 	},
 
 	// ClipboardContent.getType() callback
-	_navigatorClipboardGetTypeCallback: function(clipboardContent, blob, type) {
-		var that = this;
+	_navigatorClipboardGetTypeCallback: async function(clipboardContent, blob, type) {
 		if (type == 'image/png') {
 			this._pasteTypedBlob(type, blob);
 			return;
 		}
 
-		blob.text().then(function(text) {
-			if (type === 'text/html' && text.substring(0, 4) === '<img') {
-				// Got an image, work with that directly.
-				clipboardContent.getType('image/png').then(function(blob) {
-					that._navigatorClipboardGetTypeCallback(clipboardContent, blob, 'image/png');
-				}, function(error) {
-					window.app.console.log('clipboardContent.getType(image/png) failed: ' + error.message);
-				});
-				return;
-			}
-
-			that._navigatorClipboardTextCallback(text, type);
-		}, function(error) {
+		let text;
+		try {
+			text = await blob.text();
```

Comment:
Looks like this is because we redeclare blob further down by virtue of using the same parameter name which was previously shadowing... I've renamed it to image in the lower bit